### ISSUE TITLE
proxy task and url configuration

### ISSUE THREAD

**Issue by langecode** [Nykredit/gript#72]
After upgrading Gript version I was running the default task and get the following message.
```
[12:51:53] Starting 'proxy'...
[12:51:53] Your config is missing the mandatory url value. Refer to the readme.md and check the gulpfile.js
```

`url` is pretty general term and I really cannot seem to find out what url this is about, is the url of my application in production and why would the build setup need to know that?.Then reading the output and starting to read the gulp files I realize it is something about a proxy and it seems to maybe by the target for the traffic of the proxy. Now I did not ask for any proxy to be started up and I am not really sure why or what it should do for me. Is it a proxy for REST requests? For recording those? Or just for debugging? Or?

Anyway I think this should at the very least be somewhat more documented and the configuration parameter should probably have been more specifically named (changing this now would be a breaking change so I guess that is too late). Also is it really something to be part of the default task (guess that is also too late now though :-) ).

**Comment by langecode** [comment]
Seems this proxy thing has been part of the default task for a long time - just did not require any configuration parameters until #54 

I guess we should keep required configuration to an absolute minimum and make sure that defaults a sensible.

**Comment by jotka** [comment]
This is a target URL for the proxy - one of the historical features :) 
The reason you see the message is that in one the new versions I started to validate for the URL presence. 
I think I can make the change, so if the URL and/or port is not present, the proxy will not be started. This will be backward compatible change, because when URL and/or port is missing, it will not be started anyway.

**Comment by langecode** [comment]
Sounds good :-)

Looking through the history I did realize it has been there for quite sometime - just did not break my default target until the validation of the url configuration - think it was part of #54

**Comment by jotka** [comment]
>think it was part of #54

indeed.

**Comment by jotka** [comment]
published as 0.0.49.

Just remove `url`, `hostHeader` and `proxy` section from your `gulpfile.js` and the proxy will not be started.
